qt_ui/windows/AirWingConfigurationDialog.py

```diff
@@ -836,7 +849,7 @@ def _build_air_wing(self) -> dict:
             for s in sqs:
                 cp = s.location.at
                 if isinstance(cp, Point):
-                    key = s.location.name
+                    key = s.location.full_name
```

Comment:
I'm sure you'll ask about this, I made this change to use the full name so that carriers would save with a very deterministic name. The only potential issue with this change is that loading squadrons from a campaign with different types of carriers (ex. a faction that has a kuz and a cvn) it would fail to load. While that is less than ideal, the current implementation wouldn't let you load a air wing config at all unless it was the same exact campaign with the same exact carrier names so overall I see it as an improvement with perhaps a better solution out there that I can't come up with myself but I will defer to your judgement on that one